fla/ops/moba/naive.py

```
@@ -0,0 +1,412 @@
+# Copyright (c) 2023-2026, Songlin Yang, Yu Zhang, Zhiyuan Li
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# For a list of all contributors, visit:
+#   https://github.com/fla-org/flash-linear-attention/graphs/contributors
+
+from functools import lru_cache
```

Flake8 (7.3.0)

[error] 'functools.lru_cache' imported but unused

(F401)